content/developer/writing_widgets/widget-definition.md

```diff
@@ -140,8 +141,13 @@ fetchUrl: '[manager]/executions?is_system_workflow=false[params]'
     * `[params:param_name1(,param_name2)]` is replaced with "&paramName1:paramValue1" in the URL.
      Please note that this can be used both to selectively pick pagination parameter as well as custom parameters (see [fetchParams function](#fetchparams-widget-toolbox)).
       This mode is **exclusive** - parameters not specified explicitly are skipped.
-      When using selective param picking (`[params:param_name]`) you can use a pre-defined `gridParams` tag to include all pagination parameters (`_size`, `_offset`, `_sort`) instead of specifying explicitly  each of the three.
-
+      When using selective param picking (`[params:param_name]`) you can use a pre-defined `gridParams` tag to include [pagination parameters](https://docs.cloudify.co/api/v3.1/#pagination) (`_size`, `_offset`), [sorting parameter](https://docs.cloudify.co/api/v3.1/#sorting) (`_sort`) and search parameter (`_search`) instead of specifying explicitly each of the four. It means `fetchUrl: '[manager]/agents?[params:gridParams]'` is equivalent for 
```

Comment:
This line is quite long. Can we split it into multiple shorter lines for easier editing? (120 characters max would be best)

Reply:
Also, in this line we are using a link to a concrete version of the API (v3.1). Can we link to the page, e.g. using `relref`, so it will always be up-to-date and use the latest version? :slightly_smiling_face:

Reply:
equivalent for -> equivalent **to**

Reply:
- equivalent for fixed
- lines splitted
- added definition for https://docs.cloudify.co/api/v3.1 in the header